src/managing-domains/managing-domain-ownership.md

```diff
@@ -32,31 +36,31 @@ safeTransferFrom(address from, address to, uint256 tokenId)
 safeTransferFrom(address from, address to, uint256 tokenId, bytes _data)
 ```
 
-If one of these methods is called, both an approved operator and a Resolution address for a domain get reset.
+If one of these methods is called, both an approved operator and a Resolution address for a domain be reset.
 
 {% hint style="info" %}
-**Note:** the current implementation of transferring only resets a Resolver address, but doesn't reset records stored by a Resolver smart contract. It means that after setting a new Resolver address for a transferred domain, if the Resolver address matches the previous one, a new domain owner will get Resolution settings of a previous owner.
+**Note:** the current implementation of transferring only resets a `Resolver` address, but doesn't reset records stored by a `Resolver` smart contract. This means that after setting a new `Resolver` address for a transferred domain, if the `Resolver` address matches the previous one, a new domain owner will resolve to the `Resolution` settings of the previous owner.
```

Comment:
> a new domain owner will resolve to the

I'm not sure it's accurate to say that a domain owner (itself) will resolve to something.

Reply:
Changed to:
>**Note:** the current implementation of transferring only resets a `Resolver` address, but doesn't reset records stored by a `Resolver` smart contract. In other words, the records stored on a domain won't automatically reset whe